src/HotChocolate/Fusion/src/Core/Planning/Pipeline/ExecutionNodeBuilderMiddleware.cs

```diff
@@ -218,6 +219,8 @@ private Resolve CreateResolveNodeNode(
             requestDocument,
             executionStep.ParentSelection,
             selectionSet,
+            // TODO
```

Comment:
What about this TODO?

Reply:
Forgot to handle the case of the root `node` selection. Added a test now. I'll add a bunch more tests over the weekend where node(s) is being used and an error occurs.